File: algorithm/python/BOJ2908.py
# problem : https://www.acmicpc.net/problem/2908

import sys

A_list, B_list = map(list, sys.stdin.readline().split())

A_list.reverse()
B_list.reverse()

A = ''.join(A_list)
B = ''.join(B_list)

if int(A) > int(B):
    print(A)
else:
    print(B)

File: algorithm/typescript/BOJ2908.ts
// problem : https://www.acmicpc.net/problem/2908

import fs = require("fs");

const input: string[] = fs.readFileSync("/dev/stdin").toString().split(" ");
const A: number = Number(input[0].split("").reverse().join(""));
const B: number = Number(input[1].split("").reverse().join(""));

if (A > B) {
  console.log(A);
} else {
  console.log(B);
}
